<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1635</width>
    <height>846</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoapTyping</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../hla.qrc">
    <normaloff>:/png/images/about.png</normaloff>:/png/images/about.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">border-color: rgb(255, 170, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2" stretch="20,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="statusbarleft">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>test</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusbarright">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>test</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1635</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionLoad"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
    <addaction name="actionExport"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <widget class="QMenu" name="menuSetting">
     <property name="title">
      <string>Setting</string>
     </property>
     <property name="icon">
      <iconset resource="../hla.qrc">
       <normaloff>:/png/images/control.png</normaloff>:/png/images/control.png</iconset>
     </property>
     <addaction name="actionSet_Thread"/>
     <addaction name="actionSet_Exon_Trim"/>
    </widget>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
    <addaction name="actionForward"/>
    <addaction name="actionBackward"/>
    <addaction name="separator"/>
    <addaction name="actionApproval"/>
    <addaction name="actionReview"/>
    <addaction name="separator"/>
    <addaction name="actionAllele_Comparator"/>
    <addaction name="actionAllele_Alignment"/>
    <addaction name="actionUpdate_Database"/>
    <addaction name="separator"/>
    <addaction name="menuSetting"/>
   </widget>
   <widget class="QMenu" name="menuZoom">
    <property name="title">
     <string>Zoom</string>
    </property>
    <addaction name="actionApply_One"/>
    <addaction name="actionApply_All"/>
    <addaction name="separator"/>
    <addaction name="actionEdit_Multi"/>
    <addaction name="actionEdit_One"/>
    <addaction name="separator"/>
    <addaction name="actionAnalyze"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionHelp_Documents"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuZoom"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="ToolBarFile">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionLoad"/>
   <addaction name="actionSave"/>
   <addaction name="actionDelete"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QToolBar" name="ToolBarTool">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionReset"/>
   <addaction name="actionBackward"/>
   <addaction name="actionForward"/>
   <addaction name="actionApproval"/>
   <addaction name="actionReview"/>
   <addaction name="actionAllele_Comparator"/>
   <addaction name="actionAllele_Alignment"/>
   <addaction name="actionUpdate_Database"/>
   <addaction name="menuSetting"/>
  </widget>
  <widget class="QToolBar" name="ToolBarPeak">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionY_Range_Zoom_Increase"/>
   <addaction name="actionY_Range_Zoom_Reduce"/>
   <addaction name="actionY_Zoom_Increase"/>
   <addaction name="actionY_Zoom_Reduce"/>
   <addaction name="actionX_Zoom_Increase"/>
   <addaction name="actionX_Zoom_Reduce"/>
   <addaction name="actionReset_Zoom_Setting"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/opennewfile.png</normaloff>:/png/images/opennewfile.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/openoldfile.png</normaloff>:/png/images/openoldfile.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/saveall.png</normaloff>:/png/images/saveall.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/deleteall.png</normaloff>:/png/images/deleteall.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/report.png</normaloff>:/png/images/report.png</iconset>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/reset.png</normaloff>:/png/images/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/forward.png</normaloff>:/png/images/forward.png</iconset>
   </property>
   <property name="text">
    <string>Previous Mismatch Position</string>
   </property>
  </action>
  <action name="actionBackward">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/backward.png</normaloff>:/png/images/backward.png</iconset>
   </property>
   <property name="text">
    <string>Next Mismatch Position</string>
   </property>
  </action>
  <action name="actionApproval">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/marksuccessfulall.png</normaloff>:/png/images/marksuccessfulall.png</iconset>
   </property>
   <property name="text">
    <string>Approval</string>
   </property>
  </action>
  <action name="actionReview">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/markfinishedall.png</normaloff>:/png/images/markfinishedall.png</iconset>
   </property>
   <property name="text">
    <string>Review</string>
   </property>
  </action>
  <action name="actionAllele_Comparator">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/alignpair.png</normaloff>:/png/images/alignpair.png</iconset>
   </property>
   <property name="text">
    <string>Allele Comparator</string>
   </property>
  </action>
  <action name="actionAllele_Alignment">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/alignlab.png</normaloff>:/png/images/alignlab.png</iconset>
   </property>
   <property name="text">
    <string>Allele Alignment</string>
   </property>
  </action>
  <action name="actionUpdate_Database">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/updatedatabase.png</normaloff>:/png/images/updatedatabase.png</iconset>
   </property>
   <property name="text">
    <string>Update Database</string>
   </property>
  </action>
  <action name="actionSet_Thread">
   <property name="text">
    <string>Set Thread</string>
   </property>
  </action>
  <action name="actionSet_Exon_Trim">
   <property name="text">
    <string>Set Exon Trim</string>
   </property>
  </action>
  <action name="actionApply_One">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/apply.png</normaloff>:/png/images/apply.png</iconset>
   </property>
   <property name="text">
    <string>Apply One</string>
   </property>
  </action>
  <action name="actionApply_All">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/apply.png</normaloff>:/png/images/apply.png</iconset>
   </property>
   <property name="text">
    <string>Apply All</string>
   </property>
  </action>
  <action name="actionEdit_Multi">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/apply.png</normaloff>:/png/images/apply.png</iconset>
   </property>
   <property name="text">
    <string>Edit Multi</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionEdit_One">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/apply.png</normaloff>:/png/images/apply.png</iconset>
   </property>
   <property name="text">
    <string>Edit One</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionAnalyze">
   <property name="text">
    <string>Analyze</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionHelp_Documents">
   <property name="text">
    <string>Help Documents</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/control.png</normaloff>:/png/images/control.png</iconset>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
  <action name="actionY_Range_Zoom_Increase">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/zoomyaxisincrease.png</normaloff>:/png/images/zoomyaxisincrease.png</iconset>
   </property>
   <property name="text">
    <string>Y Range Zoom (+)</string>
   </property>
  </action>
  <action name="actionY_Range_Zoom_Reduce">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/zoomyaxisreduce.png</normaloff>:/png/images/zoomyaxisreduce.png</iconset>
   </property>
   <property name="text">
    <string>Y Range Zoom (-)</string>
   </property>
  </action>
  <action name="actionY_Zoom_Increase">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/zoomyincrease.png</normaloff>:/png/images/zoomyincrease.png</iconset>
   </property>
   <property name="text">
    <string>Y Zoom (+)</string>
   </property>
  </action>
  <action name="actionY_Zoom_Reduce">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/zoomyreduce.png</normaloff>:/png/images/zoomyreduce.png</iconset>
   </property>
   <property name="text">
    <string>Y Zoom (-)</string>
   </property>
  </action>
  <action name="actionX_Zoom_Increase">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/zoomxincrease.png</normaloff>:/png/images/zoomxincrease.png</iconset>
   </property>
   <property name="text">
    <string>X Zoom (+)</string>
   </property>
  </action>
  <action name="actionX_Zoom_Reduce">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/zoomxreduce.png</normaloff>:/png/images/zoomxreduce.png</iconset>
   </property>
   <property name="text">
    <string>X Zoom (-)</string>
   </property>
  </action>
  <action name="actionReset_Zoom_Setting">
   <property name="icon">
    <iconset resource="../hla.qrc">
     <normaloff>:/png/images/zoomreset.png</normaloff>:/png/images/zoomreset.png</iconset>
   </property>
   <property name="text">
    <string>Reset Zoom Setting</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../hla.qrc"/>
 </resources>
 <connections/>
</ui>
